users/faq-parts/general.rst

```diff
@@ -32,6 +32,8 @@ The following may be synchronized or not, depending:
 -  File permissions (when supported by file system; on Windows only the
    read only bit is synchronized)
 -  Symbolic links (synced, except on Windows, but never followed)
+   - Because a symbolic link is never followed, the synced link will
```

Comment:
This is a bit imprecise. The target doesn't need to be synced, it just need to exist, like for any other symlink.